mkdocs/commands/build.py

```diff
@@ -324,13 +340,17 @@ def build(config, live_server=False, dump_json=False, clean_site_dir=False):
     if clean_site_dir:
         log.info("Cleaning site directory")
         utils.clean_directory(config['site_dir'])
+
     if not live_server:
         log.info("Building documentation to directory: %s", config['site_dir'])
         if not clean_site_dir and site_directory_contains_stale_files(config['site_dir']):
             log.info("The directory contains stale files. Use --clean to remove them.")
 
+    # Set the status of a dirtyreload feature, a build is considered 'dirty' if clean_site_dir = False
+    dirty = not clean_site_dir
```

Comment:
Sorry I missed this before, but let's just change the `build` function's argument from `clean_site_dir=False` to `dirty=False`. After all, we are changing the default, and the consistent API will be less confusing later.